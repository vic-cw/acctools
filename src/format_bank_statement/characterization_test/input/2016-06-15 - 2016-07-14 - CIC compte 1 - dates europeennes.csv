Date d'opération,Date de valeur,Montant,Libellé,Solde
28/06/2016,28/06/2016,-700.00,PRELEVEMENT,123.00
05/07/2016,05/07/2016,40.00,VIREMENT RECU,123.00
05/07/2016,05/07/2016,-1.15,PAYPAL COMMERCE ELECTRONIQUE,123.00
