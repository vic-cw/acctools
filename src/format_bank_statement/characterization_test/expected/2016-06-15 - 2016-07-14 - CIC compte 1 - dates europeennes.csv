2016-06-28,2016-06-28,-700.00,PRELEVEMENT,123.00
2016-07-05,2016-07-05,40.00,VIREMENT RECU,123.00
2016-07-05,2016-07-05,-1.15,PAYPAL COMMERCE ELECTRONIQUE,123.00
